### DISCUSSION TITLE
Consider reducing minimumExposureDurationMinutes from 15 minutes to something lower to account new variants of COVID19

### DISCUSSION THREAD

**Discussion by LeadSongDog** [cds-snc/covid-alert-documentation#35] [Ideas] [answered]
# Summary | Résumé
Reduce minimumExposureDurationMinutes 

> One paragraph explanation of the bug, including why it is a problem (i.e. what does it prevent a user from doing or accomplishing). Screenshots are greatly helpful here too.

New variants of concern are known to propagate through briefer generations (as little as 24 hours) and on shorter contacts. Please consider reducing minimumExposureDurationMinutes from the existing arbitrary value of 15.

**Comment by obrien-j** [comment]
Hey @LeadSongDog , apologies for the shuffling. In keeping with our newly really release #31 practices, I've moved this up into the covid-alert-documentation repo as our central point for taking input from folks outside the team, and am converting the issue into a Discussion.

**Reply by LeadSongDog** [reply]
No apology needed, thank you for taking the discussion to where it belongs.

**Comment by emilykuret** [comment, marked as answer]
@LeadSongDog We're working with PHAC (Public Health Agency of Canada) and others on how we approach the variants. As soon as we know more we will share out. This is very much on our radar 🙌

**Reply by LeCanardQuoi** [reply]
How much longer will it take at HC to catch up with the UK NHS folks more experienced at responding to B.117?
“We changed the sensitivity of the exposure notification algorithm and we are going to have to make similar changes if it turns out any other variants that are emerging need an intervention”
https://www.digitalhealth.net/2021/03/nhs-covid-19-app-important-pillar-in-easing-lockdown-restrictions/

**Reply by LeadSongDog** [reply]
@emilykuret Any updates? Two months have gone by....

**Reply by CalvinRodo** [reply]
At the moment we have no more info to share on modifying minimumExposureDurationMinutes

**Comment by LeCanardQuoi** [comment]
Fyi, issue was first raised in mid December but deferred for internal discussions at CDS. Testing would have been required and impact on battery utilisation (and documentation) would need to be scrutinized.
https://github.com/cds-snc/covid-alert-app/issues/1304

**Comment by LeCanardQuoi** [comment]
The Germans are providing for 5 minutes proximity contact duration and improved risk assessment on their corona-warn-app in response to the VOC.
See https://www.coronawarn.app/en/blog/2021-02-23-corona-warn-app-risk-calculation-optimization/
and most recent:
https://www.coronawarn.app/en/blog/2021-03-19-risk-calculation-improvement/
